HW5/KaziutaRostyslav/hw5.py

```diff
@@ -0,0 +1,41 @@
+# list_num = [1,2,3,4,5,6,7,8,"9","10"]
+# float_list = []
+# for i in list_num:
+#     if isinstance(i,int):
+#         float_list.append(float(i))
+#
+# print(float_list)
+
+
+
```

Comment:
good job
look at this version of code
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n},\n {fibonacci_numbers}')